test/api/v3/integration/groups/POST-groups_id_removeMember.test.js

```diff
@@ -129,6 +132,11 @@ describe('POST /groups/:groupId/removeMember/:memberId', () => {
       expect(invitedUserWithoutInvite.invitations.party).to.be.empty;
     });
 
+    it('removes new messages from a member who is removed', async () => {
+      await partyLeader.post(`/groups/${party._id}/removeMember/${removedMember._id}`);
+      expect(removedMember.newMessages).to.be.empty;
```

Comment:
Let's move the send message logic here. 

You'll need to use the sync method to bring this member up to date. You're asserting on the version of the data before the message was sent.